app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::GreetingsController < ApplicationController
+  def random_greeting
+    @greetings = Message.pluck(:greeting)
+    @greeting = @greetings.sample
+    render json: { greeting: @greeting }
+  end
+end
+
+# module Api
+#   module V1
+#     class GreetingsController < ApplicationController
+#       def random_greeting
+#         @greetings = Message.pluck(:greeting)
+#         @greeting = @greetings.sample
+#         render json: { greeting: @greeting }
+#       end
+#     end
+#   end
+# end
```

Comment:
[OPTIONAL BUT RECOMMENDED] Please remove commented code from your codebase because it makes your codebase messy if you want to check your previous code please check your GitHub commit history :slightly_smiling_face:

Reply:
I just forgot to remove it. So, I removed it now.